### DISCUSSION TITLE
Hydration errors when using input type color but not when using input type text

### DISCUSSION THREAD

**Discussion by sonJ9** [vercel/next.js#41522] [Help] [answered]
Next.js shows 3 Unhandled Runtime Errors including hydration error when I use `<input type="color"/>` but doesn't show any error when I use `<input type="text"/>`

Can someone help me fix this issue?

**Comment by icyJoseph** [comment, marked as answer]
Hi,

The hydration error comes from React's attempt at hydrating the page. I've tried to reproduce this but no luck, could you share more on what is it your are doing? Perhaps you've got a repository? or Stackblitz link?

Are you perhaps using tailwind of something like that, just out of curiosity.

**Reply by sonJ9** [reply]
Hey,  thanks for the reply. 

I am trying to build a color & gradient generator using **_NextJs_** & **_Sass_**. This is my first time using Next (I used to use React before) and I kept on getting errors while using `<input type="color"/>` 

This is the code in question: 
[Code](https://goonlinetools.com/snapshot/code/#2hwqmd6lh4c9dsv16vniu)

And these are the 3 errors I get when I switch **type="text"** to **type="color"**: 
[Error 1](https://imgur.com/zIyWUer)
[Error 2](https://imgur.com/vdEciLa)
[Error 3](https://imgur.com/U6AAHit)

**Reply by icyJoseph** [reply]
I am running that code, albeit with `css` Modules and it works fine. Is there anything else going on your application? 

Do you think you can invite me to the project, or show a bit more of what's going on? 

This issue happens because the HTML sent by the server, is not matching what React rebuilds (hydrates) on the client. Got any scripts targeting type color inputs?

**Reply by sonJ9** [reply]
Can't believe it's working on your application but not on mine. 

I've just sent you an invite. Hope you can figure out the fix for it :))

**Reply by icyJoseph** [reply]
Hi!

Well, still nothing... it works. I changed your inputs to color and it works alright!

![Screenshot 2022-10-18 at 20 53 05](https://user-images.githubusercontent.com/21013447/196518904-01520d78-a1eb-41bf-ad61-c2549983cca9.png)


I wonder if you see the issue if you run the application in incognito mode?

**Reply by icyJoseph** [reply]
I've figure out that there's a chrome extension, which in Windows OS triggers this error, could you please confirm if it works in Incognito Mode? And if so, which extensions have you gotten installed?

**Reply by sonJ9** [reply]
Hey! I just checked it in incognito mode and it somehow worked without errors. 

You were right, it was because of a chrome extension (**_ColorPick Eyedropper_**) that I had ON. I disabled it and now it works without errors in normal mode as well. 

Can't thank you enough. Hope you have a great day, brother!! :)))
